### DISCUSSION TITLE
Error on npm run test: Process completed with exit code 1

### DISCUSSION THREAD

**Discussion by marcos-de-barros-vioque** [neuefische/questions#339] [Web] [answered]
This error occurs while doing "npm run test". 
The error message points at the imported built-in component (<Navbar /> in the case, but the navigation looks fine):

![Bildschirmfoto 2022-11-29 um 16 05 01](https://user-images.githubusercontent.com/115008408/204565177-c6a1e392-8d5c-4e54-a3e1-49c127782ae1.png)

This is the link to my pull request: https://github.com/marcos-de-barros-vioque/capstone/pull/8

**Comment by actyralf** [comment]
Could you please link your pull request to the issue?

**Reply by marcos-de-barros-vioque** [reply]
Hi @actyralf I've just added the link to the PR: https://github.com/marcos-de-barros-vioque/capstone/pull/8
Thx

**Comment by actyralf** [comment, marked as answer]
You do a named export in `/components/Layout` then you have to do a named import as well: `import {Layout} from "../components/Layout"` and render the `<Layout>` component.

**Reply by marcos-de-barros-vioque** [reply]
All checks have passed! Thx a bunch @actyralf
